src/ContaoManager/Plugin.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * This file is part of contao-community-alliance/event-dispatcher.
+ *
+ * (c) 2013-2017 Contao Community Alliance <https://c-c-a.org>
+ *
+ * This project is provided in good faith and hope to be usable by anyone.
+ *
+ * @package    contao-community-alliance/event-dispatcher
+ * @author     Sven Baumann <baumann.sv@gmail.com>
+ * @copyright  2013-2017 Contao Community Alliance <https://c-c-a.org>
+ * @license    https://github.com/contao-community-alliance/event-dispatcher/LICENSE LGPL-3.0+
+ * @link       https://github.com/contao-community-alliance/event-dispatcher
+ * @filesource
+ */
+
+namespace ContaoCommunityAlliance\Contao\EventDispatcher\ContaoManager;
+
+use Contao\CoreBundle\ContaoCoreBundle;
+use Contao\ManagerBundle\ContaoManagerBundle;
+use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
+use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
+use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
+use ContaoCommunityAlliance\Contao\EventDispatcher\CcaEventDispatcherBundle;
+use DependencyInjection\Container\CcaDependencyInjectionBundle;
+
+/**
+ * Plugin for the Contao Manager.
+ */
+class Plugin implements BundlePluginInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getBundles(ParserInterface $parser)
+    {
+        return [
+            BundleConfig::create(CcaEventDispatcherBundle::class)
+                ->setLoadAfter(
+                    [
+                        ContaoCoreBundle::class,
+                        ContaoManagerBundle::class,
```

Comment:
This plugin loads if installed contao managed edition.
The managed edition show for [configuration](https://github.com/contao-community-alliance/event-dispatcher/blob/feature/contao4/composer.json#L56) in composer.json.